projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,72 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: 1.5rem;
+    justify-content: center;
+    overflow: hidden;
+
+    > button {
+        .button-base();
+
+        position: relative;
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        flex-direction: column;
+        padding: 1.125rem 0.25rem 1.125rem 0.25rem;
+        gap: 0.5rem;
+        height: auto;
+        font-size: 0.8125rem;
+        white-space: pre-line;
+        min-width: 7.125rem;
```

Comment:
Do we need `height`, `min-width` and `font-weight`?

Reply:
without min-width
<img width="308" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/bb4ef234-ed2b-45b6-8453-acb8d844e32b">

Reply:
That's ok, we should just add width to the container, that is where it should come from.